aws-sign-web.js

```diff
@@ -248,6 +248,42 @@
         }
     }
 
+    /**
```

Comment:
This comment could be shortened in my opinion.
What I don't like too much is that it seems to be copied 1-by-1 from the AWS documentation.
One-line sentence and the link to the reference would be enough I guess.